src/config/entities/__tests__/configuration.ts

```diff
@@ -24,6 +24,10 @@ export default (): ReturnType<typeof configuration> => ({
       token: faker.number.int(),
     },
   },
+  features: {
+    humanDescription: true,
+    messagesCache: false,
```

Comment:
OOC: why is this disabled in the tests?

Reply:
This is just set because the test configuration needs to follow the `configuration.ts` interface, but it's not used by unit tests (besides the ones which specifically set it, like `transaction-api.service.spec.ts`, so I picked a boolean value. Maybe it's more accurate if we use `faker.datatype.boolean()` also here, wdyt?

Note: It could have an influence on e2e tests, but we don't have any for messages. In that case, it should be set in `e2e-setup.ts` if I'm not wrong.

Reply:
This file represents the test configuration and since this is a feature flag that is bound to be deleted once the releases are synced I think it makes sense for it to be `true`.

If tests are written that target messages we should assume tests that handle caching on the messages side.

I don't think `faker.datatype.boolean()` is a good choice for this feature flag in this case as each test should assume this new standard. I think it'd make sense to have it `false` if the service assumed no caching for the messages AND we'd have tests targeting when the feature is enabled (there we'd set it to `true`).

Reply:
Yes, makes perfect sense, I agree.

I've updated it in [e624337](https://github.com/safe-global/safe-client-gateway-nest/pull/654/commits/e6243371ad9238adaf8ed44d678134365d6a3608), thanks!